docs/wiki/Home.md

```diff
@@ -1,8 +1,8 @@
-<img src="images/logo_wide_v1.svg" width="250">
+# DCRDEX WIKI
```

Comment:
Hmm, the logo was nice although the inline html is awkward.  It's why I added the `MD033` exception.  
Let's use what we do in the README:

`# <img src="docs/images/logo_wide_v1.svg" alt="DCRDEX" width="250">`